### ISSUE TITLE
Error on custom field in Reports module when using mysql 5.7+ or postgres

### ISSUE THREAD

**Issue by claytongulick** [freescout-helpdesk/freescout#1933]
In ReportsServiceProvider.php:

```
89                 $custom_fields = \Modules\CustomFields\Entities\CustomField::whereIn('mailbox_id', $mailbox_ids)
90                     ->groupBy('name')
91                     ->get();
```

throws an error because of the use of the deprecated and non-standard groupBy without aggregates in the select list.

I manually modified the file to comment out the groupBy on line 90, and the reports page seems to be functioning properly.

**Comment by freescout-helpdesk** [comment]
What database are you using?

**Comment by freescout-helpdesk** [comment]
Fixed in Reports Module v1.0.3
